src/sgame/sg_bot_nav.cpp

```diff
@@ -830,17 +830,37 @@ Global Bot Navigation
 =========================
 */
 
+static constexpr int WALLCLIMB_MAX_TIME = 3000;
```

Comment:
I'd be curious to play with variable to see how useful this mechanism is. Could it be possible to have it as as cvar?

Reply:
We can do that later. I am already guilty of adding so many cvars that it is becoming increasingly difficult to set them up.